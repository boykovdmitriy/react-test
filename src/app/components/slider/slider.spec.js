import React from 'react';
import {WrappedSlider} from './index'

describe('wrapped slider component', () => {
    it('render wrapped slider component', () => {
        const wrapper = shallow(
            <WrappedSlider>
                <div>1</div>
                <div>2</div>
                <div>3</div>
            </WrappedSlider>
        );
        expect(wrapper).toMatchSnapshot();
    });
    it('render wrapped slider and check rendered content', () => {
        const wrapper = shallow(
            <WrappedSlider>
                <div>1</div>
                <div>2</div>
                <div>3</div>
            </WrappedSlider>
        );
        expect(wrapper.contains([<div>1</div>, <div>2</div>, <div>3</div>])).toEqual(true);
    });
});