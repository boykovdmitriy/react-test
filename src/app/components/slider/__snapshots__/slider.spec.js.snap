// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wrapped slider component render wrapped slider component 1`] = `
<Carousel
  autoPlay={true}
  axis="horizontal"
  centerMode={true}
  centerSlidePercentage={80}
  dynamicHeight={false}
  emulateTouch={false}
  infiniteLoop={true}
  interval={1000}
  onChange={[Function]}
  onClickItem={[Function]}
  onClickThumb={[Function]}
  selectedItem={0}
  showArrows={true}
  showIndicators={false}
  showStatus={false}
  showThumbs={false}
  statusFormatter={[Function]}
  stopOnHover={true}
  swipeScrollTolerance={5}
  transitionTime={350}
  useKeyboardArrows={false}
  verticalSwipe="standard"
  width="100%"
>
  <div>
    1
  </div>
  <div>
    2
  </div>
  <div>
    3
  </div>
</Carousel>
`;
